src/main/java/frc/robot/hardware/motor/PIDAble.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.hardware.motor;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.request.angle.IAngleRequest;
+import frc.robot.hardware.request.value.IValueRequest;
+import frc.utils.calibration.sysid.SysIdCalibrator;
+
+public interface PIDAble {
```

Comment:
be consistent. its either
IMotor and IPID 
or
MotorAble (or smth) and PIDAble

Reply:
its consistent, the unterface represent what it does.
imotor implements a motor, so ipid should implement pid. Butt its not the case so Pidable represent something that can do pid